README.md

```diff
@@ -57,6 +58,22 @@ client = Bitreserve::Client.new
 puts client.all_tickers
 ```
 
+# Options
+
+This is a summary of the supported options when instantiating a new client, and their default values:
+
+```ruby
+Bitreserve::Client.new(
+  # bearer_token for OAuth authentication
+  token: ENV['BITRESERVE_AUTH_TOKEN']
+
+  # use the sandbox api (https://api-sandbox.bitreserve.org) instead of the real one
+  sandbox: false
```

Comment:
Have you thought of making this a global config, and not a per client config? What would be the pros and cons?